packages/frontend/components/global/CookiesBanner/CookiesBanner.styles.ts

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+
+export const Banner = styled.div`
+  padding-block: ${pxToRem(36)};
+`;
+
+export const Content = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
For now, allow this flex area to wrap so the buttons are below when we run out of space. We'll refine this as Lael designs the mobile layouts. 😄